src/styles/utils/_variables.scss

```
@@ -0,0 +1,15 @@
+$cellSize: 100px;
+$calendarWidth: ($cellSize * 7) + (1 * 6);
+$daysOfWeek:
+  "mon" 0,
+  "tue" $cellSize + 1,
```

you can calc it just one time instead calc each time
```suggestion
$calendarWidth: ($cellSize * 7) + (1 * 6);
$day-margin: $cellSize + $cellGap;
$daysOfWeek:
  "mon" 0,
  "tue" $day-margin,
```